src/parameters/Parameters.jl

```diff
@@ -74,6 +74,9 @@ Base.@kwdef struct ClimaAtmosParameters{FT, TP, RP, IP, MPP, WP, SFP, TCP} <:
     zd_viscous::FT
     zd_rayleigh::FT
     kappa_2_sponge::FT
+    # Radiation
+    ocean_surface_albedo::FT
```

Comment:
It could be more clear to call this "idealized_ocean_albedo".

Reply:
Sounds good, I will add this to the next ClimaParams breaking release.